src/cdk/generator.ts

```diff
@@ -25,6 +27,11 @@ export interface ResourceGeneratorProps {
    * The {@link HttpApi} to attach routes to.
    */
   httpApi: HttpApi
+
+  /**
+   * Default Lambda function options.
+   */
+  functionOptions?: NodejsFunctionProps
```

Comment:
Will we be able to get the idea of what the usual options are by looking at the interface or it's generic and we need some examples of what can be passed there?

Reply:
You can see them here https://jetbridge.github.io/jetkit-cdk/interfaces/iapiviewclassmetadata.html